gist/gist.py

```diff
@@ -138,19 +138,6 @@ def __init__(self, token, editor=None):
         """
         self.token = token
         self.editor = editor
-        self.proxies = {}
```

Comment:
I am going through the requests code to better understand this. I think I may be mistaken about how the proxy information is populated. I still want to confirm that this change will not cause a regression.